tests/microos/set_scc_proxy_url.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register the already installed system on a specific SCC server/proxy if needed
+# Maintainer: qa-c@suse.de
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi qw(
+    enter_cmd
+    get_var
+    save_screenshot
+);
+
+sub run {
+    my $self = shift;
+    set_scc_proxy_url();
+}
+
+sub set_scc_proxy_url {
```

Comment:
Why not include the function from lib/migration.pm? We shouldn't duplicate code here.

Reply:
Do you mean/prefer to move the _sub_ inside `lib/migration.pm`?

Reply:
No, this function already defined in lib/migration.pm, you just need include it and invoke it in your code.

use migration 'set_scc_proxy_url';

Reply:
Done